stable/postgresql/README.md

```diff
@@ -116,6 +116,7 @@ The following tables lists the configurable parameters of the PostgreSQL chart a
 | `service.loadBalancerIP`                      | loadBalancerIP if service type is `LoadBalancer`                                                                                                                          | `nil`                                                         |
 | `service.loadBalancerSourceRanges`            | Address that are allowed when svc is LoadBalancer                                                                                                                         | []                                                            |
 | `schedulerName`                               | Name of the k8s scheduler (other than default)                                                                                                                            | `nil`                                                         |
+| `enableShmVolume`                             | Enable emptyDir volume for /dev/shm                                                                                                                                       | `true`                                                        |
```

Comment:
Is it okay for you to have, like persistence, one global value to enable for both master and slaves?

Reply:
Yes, but please add that information to the Description:
```
Enable emptyDir volume for /dev/shm both on master and slaves replicas
```